sample-notificator-domain/src/main/groovy/life/qbic/business/subscription/Subscriber.groovy

```diff
@@ -0,0 +1,34 @@
+package life.qbic.business.subscription
+
+import life.qbic.datamodel.samples.Status
+
+/**
+ * <h1>A person that subscribed to the notification table</h1>
+ *
+ * <p>A subscriber is described by his full name and an email address</p>
+ *
+ * @since 1.0.0
+ *
+*/
+class Subscriber {
+    final String firstName
+    final String lastName
+    final String email
```

Comment:
we need to know the project he/she subscribed to. Or better think about it the other way around. We could also create a Project data object that holds information on all the subscribers and the changes within this project

Reply:
from the list of subscriptions you can refer the project. Its within the sample code